test_framework/terraform/aws/sles/main.tf

```diff
@@ -250,3 +250,8 @@ resource "aws_eip" "lh_aws_eip_controlplane" {
   count    = var.lh_aws_instance_count_controlplane
   vpc      = true
 }
+
+resource "random_password" "cluster_secret" {
+  length = 64
+  special = false
+}
```

Comment:
Could you move this under the below resource call, and add a comment (ex: `Create a random string for the cluster secret`) for it. 

```hcl
resource "random_string" "random_suffix" {
...
}

<-- add here
```

Reply:
Hi @innobead ,

I moved the code and added comment. And apply those changes on all distro for review, thank you.